app/controllers/v1/inventory_invitables_controller.rb

```diff
@@ -0,0 +1,21 @@
+class V1::InventoryInvitablesController < ApplicationController
+  before_action :authenticate_user!
+  authorize_actions_for :inventory
+
+  authority_actions index: 'read'
+  def index
+    member_users = inventory.members.select(:user_id).map(&:user_id)
+    @users = User.includes(:districts)
```

Comment:
Is there a reason we can't use `joins`?  Unless there's some business logic constraint I believe we should, considering the query efficiency.

I ran this on staging with some sample data and saw these numbers.  The first query plan shows the `:includes`, and the second one is with `:joins`.

```
                                      QUERY PLAN
----------------------------------------------------------------------------------------
 Nested Loop  (cost=0.11..12.90 rows=1 width=450)
   ->  Nested Loop  (cost=0.06..8.84 rows=1 width=252)
         ->  Seq Scan on districts_users  (cost=0.00..4.77 rows=1 width=8)
               Filter: (district_id = 12)
         ->  Index Scan using users_pkey on users  (cost=0.06..4.06 rows=1 width=248)
               Index Cond: (id = districts_users.user_id)
               Filter: ((id <> 61) AND ((role)::text <> 'network_partner'::text))
   ->  Index Scan using districts_pkey on districts  (cost=0.06..4.06 rows=1 width=202)
         Index Cond: (id = 12)
(9 rows)

                                        QUERY PLAN
-------------------------------------------------------------------------------------------
 Nested Loop  (cost=0.11..10.90 rows=1 width=248)
   ->  Nested Loop  (cost=0.06..8.84 rows=1 width=252)
         ->  Seq Scan on districts_users  (cost=0.00..4.77 rows=1 width=8)
               Filter: (district_id = 12)
         ->  Index Scan using users_pkey on users  (cost=0.06..4.06 rows=1 width=248)
               Index Cond: (id = districts_users.user_id)
               Filter: ((id <> 61) AND ((role)::text <> 'network_partner'::text))
   ->  Index Only Scan using districts_pkey on districts  (cost=0.06..2.06 rows=1 width=4)
         Index Cond: (id = 12)
(9 rows)
```